x/incentives/keeper/grpc_query.go

```diff
@@ -46,7 +47,12 @@ func (k Keeper) Gauges(goCtx context.Context, req *types.GaugesRequest) (*types.
 	valStore := prefix.NewStore(store, types.KeyPrefixGauges)
 
 	pageRes, err := query.FilteredPaginate(valStore, req.Pagination, func(key []byte, value []byte, accumulate bool) (bool, error) {
-		newGauges, err := k.GetGaugeFromIDs(ctx, value)
+		gaugeIDs := []uint64{}
+		err := json.Unmarshal(value, &gaugeIDs)
+		if err != nil {
+			panic(err)
+		}
```

Comment:
And this is an example of a place we'd use the `getGaugeFromIDJsonBytes`

Reply:
Done in https://github.com/osmosis-labs/osmosis/pull/572/commits/cbabd871a117231d1e3db4a0257d46d555c098de.